### DISCUSSION TITLE
Multi level choice input for workflow

### DISCUSSION THREAD

**Discussion by yakirm-cr** [community/community#20315] [Actions]
Currently, GitHub Actions supports defining a single level of choices as input for a workflow in the following syntax:

```yaml
on:
  workflow_dispatch:
    inputs:
      logLevel:
        description: 'Log level'
        required: true
        default: 'warning' 
        type: choice
        options:
        - info
        - warning
        - debug 
```

Is it possible to support also multi-level choice input?
For example:
```yaml
on:
  workflow_dispatch:
    inputs:
      logLevel:
        description: 'Log level'
        required: true
        default: [ 'warning', 'all' ] 
        type: choice
        options:
        - info:
          - all
          - package1
        - warning:
          - all
          - package2
        - debug:
          - all
          - package3
```

In such scenario two drop-downs will be presented to the user and values listed in the 2<sup>nd</sup> drop-down will be based on the value selected in the first drop-down.

The actual value in the context 'intputs.logLevel' variable will be an array (as shown in the default value above).
What do you think?

**Comment by umassahu** [comment]
@github-staff  please have a look and suggest , we need to have multi select capaability to run multiple test suites based on the inputs
